std/concurrency.d

```diff
@@ -1716,15 +1716,7 @@ void yield(T)(T value)
 }
 
 
-version (Win64)
-{
-    // fibers are broken on Win64
-}
-else version (Win32)
-{
-    // fibers are broken in Win32 under server 2012: bug 13821
-}
-else unittest
+unittest
 {
```

Comment:
I'm not sure that bug 13821 is fixed. If it is, the bug should be closed. I'm not sure if Brad moved the tester to 2012. Otherwise, this should be left the way it was.